### PR TITLE
Feat single playlist page

### PR DESCRIPTION
**Pull request by JowelTisso**
This PR is for single playlist screen and logic:
* Small PR

### CODE REVIEW

**Review by rahulmistry751 — APPROVED** [review]
Code looks clean.

**Review by saketkum11 — APPROVED** [review]
- clean code
- much understandable

File: src/src/pages/playlist/singlePlaylist/SinglePlaylist.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import "./SinglePlaylist.css";
+import React, { useEffect, useState } from "react";
+import { Link, useParams } from "react-router-dom";
+import HistoryCard from "../../../pages/history/components/HistoryCard";
+import { useVideo } from "../../../context/provider/VideoProvider";
```
Comment (manismk) [inline src/src/pages/playlist/singlePlaylist/SinglePlaylist.jsx:5]: you can use barrel export on these files

**Review by manismk — APPROVED** [review]
Good naming convention.
small PR.
Looks Good

Remove the empty css file.

File: src/src/pages/playlist/singlePlaylist/SinglePlaylist.jsx (hunk 1 above)
Comment (JowelTisso) [reply src/src/pages/playlist/singlePlaylist/SinglePlaylist.jsx:5]: Ok

**Pull request merged**